database-plugins/src/main/java/co/cask/hydrator/plugin/DBConfig.java

```diff
@@ -62,4 +62,9 @@
   public DBConfig() {
     jdbcPluginType = "jdbc";
   }
+
+  protected String cleanQuery(String query) {
+    query = query.trim();
+    return query.endsWith(";") ? query.substring(0, query.length() - 1) : query;
```

Comment:
eh might as well handle multiple semi-colons.  Not sure what you mean by set of characters though?